learn/cookbooks/vercel.mdx

```diff
@@ -0,0 +1,65 @@
+
+
+Select the project of your choice and then click on **Deploy**. Once the deployment is complete, you will be able to go back to your project’s dashboard. 
+
+Go to the project settings tab, and then click on **Integrations**, on the left-side menu.
+
+![Selecting the integration tab in the project settings](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/02.project-settings-integrations-tab.png)
+
+Search for the [Meilisearch Cloud integration](https://vercel.com/integrations/meilisearch) in the search bar. The integration page should look like this. Click on the **Add integration** button.
+
+![Meilisearch Cloud integration page in Vercel's marketplace](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/03.meilisearch-cloud-integration-page-in-marketplace.png)
+
+Select the Vercel account or team and the project you want to add the integration to. You have the flexibility to add it to all your projects or select specific ones (one or multiple projects).
+
+![Form to add Meilisearch Cloud integration, the "Specific projects" option is selected](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/04.add-meilisearch-cloud-form.png)
+
+Click on **Continue**. It will display the list of rights the integration needs to work properly, you can then click on **Add Integration**.
+
+Next, it will take you to the Meilisearch Cloud page. Log in or create an account. New accounts enjoy a 14-day free trial period.
+
+You can choose an existing project or create a new one. To do so, complete the form with the project name and the region.
+
+![Meilisearch Cloud form to create a project complete, with "search-app" as the project's name and "Frankfurt" as the region](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/05.create-a-meilisearch-cloud-project-form.png)
+
+Once you click on **Create project**, you should see the following message:
+“Your Meilisearch + Vercel integration is one click away from being completed.” and a button to finish the setup. On clicking the button, it should redirect you to the integration page in your Vercel Dashboard.
+
+![Meilisearch Cloud integration page in Vercel's dashboard](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/06.meilisearch-cloud-integration-page-in-vercel-dashboard.png)
+
+Meilisearch creates two default API keys: [`Default Search API Key` and `Default Admin API Key`](/learn/security/master_api_keys#using-default-api-keys-for-authorization). With the `Default Admin API Key`, you can control who can access or create new documents, indexes, or change the configuration. The `Default Search API Key` can only be used to access the [search route](/reference/api/search). It is the one you want to use in your front end.
+
+Both keys are automatically added to Vercel along with the Meilisearch host or URL.
```

Comment:
Is there a difference between host and URL here? If not, I recommend using just one.